templates/scenarios/ts/sso-tab/teamsapp.yml.tpl

```diff
@@ -26,15 +26,32 @@ provision:
     writeToEnvironmentFile: # Write the information of created resources into environment file for the specified environment variable(s).
       teamsAppId: TEAMS_APP_ID
 
-  - uses: arm/deploy # Deploy given ARM templates parallelly.
-    with:
-      subscriptionId: ${{AZURE_SUBSCRIPTION_ID}} # The AZURE_SUBSCRIPTION_ID is a built-in environment variable. TeamsFx will ask you select one subscription if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select subscription if it's empty in this case.
-      resourceGroupName: ${{AZURE_RESOURCE_GROUP_NAME}} # The AZURE_RESOURCE_GROUP_NAME is a built-in environment variable. TeamsFx will ask you to select or create one resource group if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select or create resource grouop if it's empty in this case.
+  - uses: arm/deploy  # Deploy given ARM templates parallelly.
+    with:
+      # The AZURE_SUBSCRIPTION_ID is a built-in environment variable.
+      # TeamsFx will ask you select one subscription if its value is empty.
+      # You're free to reference other environment varialbe here,
+      # but TeamsFx will not ask you to select subscription
+      # if it's empty in this case.
```

Comment:
AZURE_SUBSCRIPTION_ID is a built-in environment variable, if its value is empty, TeamsFx will prompt you to select a subscription. Referencing other environment variables with empty values will skip the subscription selection prompt.